nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/about-dialog/about-dialog.component.html

```diff
@@ -0,0 +1,50 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<div class="about-dialog" tabindex="0">
+    <h2 mat-dialog-title>About</h2>
+    <mat-dialog-content>
+        <div class="flex flex-col justify-between gap-y-5">
+            @if (about$ | async; as about) {
+                <div class="flex justify-center">
+                    <img ngSrc="assets/icons/nifi-logo-about.svg" priority width="300" height="128" alt="NiFi Logo" />
+                </div>
+                <div class="flex flex-col gap-y-1">
+                    <div class="bold">{{ about.version }}</div>
```

Comment:
`bold` is not a defined class. I think you mean to use `font-bold` or change the dom element to `<strong>`